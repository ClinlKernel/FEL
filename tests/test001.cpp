#include <fstream>

int main(int argc, char** argv)
{
	std::ofstream{argv[1]}<<"{\"pass\":1,\"fail\":0}";
	return 0;
}