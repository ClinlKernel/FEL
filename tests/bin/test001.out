{"pass":1,"fail":0}